internal/mock/sqladmin.go

```diff
@@ -29,9 +31,13 @@ import (
 	"sync"
 	"time"
 
+	"google.golang.org/api/option"
 	sqladmin "google.golang.org/api/sqladmin/v1beta4"
 )
 
+// RSAKey is used for test only.
+var RSAKey = genRSAKey()
```

Comment:
nit: does this make the most sense in the mock? Or should it be in the test package?

Reply:
It looks like we'll only be using this in the `internal/cloudsql` tests, so I'll move this key there and we can always move it to a better location if necessary.